app/views/navbar.scala.html

```diff
@@ -79,19 +79,22 @@
                         <a class="navbar-button navbar-start-btn" href="@routes.AuditController.audit()">@Messages("navbar.explore")</a>
                     </li>
                 }
+
```

Comment:
I don't think any of these added lines do anything either. Not sure if you added them on purpose or if they somehow got added because of a merge